cores/esp8266/HardwareSerial.cpp

```diff
@@ -152,6 +152,9 @@ void HardwareSerial::flush()
     }
 
     uart_wait_tx_empty(_uart);
+    //Workaround for a bug in serial not actually being finished yet
+    //Wait for 8 data bits, 1 parity and 2 stop bits, just in case
+    delayMicroseconds(1100000 / uart_get_baudrate(_uart) + 1);

```

Comment:
Your original comment in the issue said this worked:
delayMicroseconds((int)(1000000.0 / uart_get_baudrate(_uart) * 11 + 1))

There is a difference of a factor of 10. Which is correct?

Reply:
1000000 * 11 == 11000000. Nuff said.

Reply:
Yes, but 1100000 != 11000000
I think your PR is missing a zero.

Reply:
Yeah, good catch. I don't know how that did happen.